erpnext_shopify/sync_products.py

```diff
@@ -423,7 +423,7 @@ def update_item_stock_qty():
 		update_item_stock(item.item_code, shopify_settings)
 
 def update_item_stock(item_code, shopify_settings, bin=None):
-	item = frappe.get_doc("Item", item_code)
+	item = frappe.get_doc("Item", {"item_code": item_code})
```

Comment:
@anandpdoshi 

While syncing from shopify we keep Item Code and Name name as shopify id. 

but while pushing data from ERPNext to Shopify, I am using filter here in case if item code is not name.